rootfs-builder/README.md

```diff
@@ -1,12 +1,12 @@
 * [Supported base OSs](#supported-base-oss)
 * [Creating a rootfs](#creating-a-rootfs)
-* [Creating a rootfs with kernel modules](#creating-a-rootfs-with-kenrel-modules)
-* [Build a rootfs using Docker*](#build-a-rootfs-using-docker*)
+* [Creating a rootfs with kernel modules](#creating-a-rootfs-with-kernel-modules)
+* [Build a rootfs using Docker*](#build-a-rootfs-using-docker)
 * [Adding support for a new guest OS](#adding-support-for-a-new-guest-os)
     * [Create template files](#create-template-files)
     * [Modify template files](#modify-template-files)
     * [Expected rootfs directory content](#expected-rootfs-directory-content)
-    * [(optional) Customise the rootfs](#(optional)-customise-the-rootfs)
+    * [optional - Customise the rootfs](#optional-customise-the-rootfs)
```

Comment:
Do you mind capitalizing "optional" (here and below)?

Reply:
Done.